django-stubs/contrib/messages/views.pyi

```diff
@@ -1,8 +1,12 @@
+from typing import Generic, TypeVar
+
 from django.forms.forms import BaseForm
 from django.http.response import HttpResponse
 from django.utils.functional import _StrOrPromise
 
-class SuccessMessageMixin:
+_F = TypeVar("_F", bound=BaseForm)
+
+class SuccessMessageMixin(Generic[_F]):
```

Comment:
Since you made this generic, it needs to be added to `django-stubs-ext` monkeypatching: `_need_generic` in `ext/django_stubs_ext/patch.py` (otherwise `SuccessMessageMixin[MyForm]` won't be allowed)